### ISSUE TITLE
Specify the rotation increment 

### ISSUE THREAD

**Issue by jil24** [openseadragon/openseadragon#1518]
While we can programmatically set arbitrary rotations or use gestures, the button-based rotation is limited to 90 degree increments.

I could replace the buttons completely with custom ones, I think it would be a lot simpler to be able to specify the increment for the built-in rotation buttons; e.g. 30 degrees would be best for my particular application.

**Comment by iangilman** [comment]
Good idea! Would you be up for making a patch to this effect? I figure it would be an option you pass into the viewer creation and it would affect both the buttons and the keys.

**Comment by jil24** [comment]
Here's my patch against the latest release version
[rotationincrement.txt](https://github.com/openseadragon/openseadragon/files/2306755/rotationincrement.txt)

.

**Comment by iangilman** [comment]
Looks fantastic! Can you make a pull request with this?

**Comment by jil24** [comment]
done; #1524
